appserver/tests/application/src/main/java/org/glassfish/main/test/app/concurrency/executor/ManagedExecutorDefinitionServlet.java

```diff
@@ -94,7 +94,7 @@ private void copyCompletableFuture(int iContextValue) {
                     "Completation stage that is supplied to copy must not be modified by the ManagedExecutorService.");
             }
 
-            final String result = stage3.get(10, TimeUnit.SECONDS);
+            final String result = stage3.get(1000, TimeUnit.SECONDS);
```

Comment:
Why one thousand seconds? That is too much ...

Reply:
I’ll revert it. I needed this for debugging and committed it by accident.

Reply:
Reverted with the last commit.